src/index.tsx

```diff
@@ -240,170 +220,172 @@ export const InOutTextarea: FC<Props> = props => {
   } = props;
 
   return (
-    <ConvertCard>
-      <CaseBar>
-        <LeftCaseBar>
-          <OptionsContainer>
-            {inOptions
-              .sort(a => {
-                if (a.active) return -1;
-                return 0;
-              })
-              .map(option => {
+    <>
```

Comment:
I think your local branch was not up to date. Could you merge the master branch from this repository into your branch? Here is a guide on [how to do it](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/syncing-a-fork).